### PR TITLE
Correct Cache API data.

### PR DESCRIPTION
**Pull request by jpmedley**
@Elchi3 Here's my cache update.

### CODE REVIEW

File: api/Cache.json [hunk 1]
```diff
@@ -4,57 +4,57 @@
       "__compat": {
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/Cache",
         "support": {
+          "webview_android": {
+            "version_added": "43",
+            "notes": [
+              "From 40 to 42, this was only available on service workers."
+            ]
```
Comment (chrisdavidmills) [inline api/Cache.json]: Seems a bit strange to have all notes in arrays, even if there is only one note. Make these string types?

File: api/Cache.json [hunk 2]
```diff
@@ -4,57 +4,57 @@
...
+              "From 27 to 29, this was only available on service workers."
             ]
           },
-          "chrome_android": {
-            "version_added": "40",
+          "opera_android": {
+            "version_added": "30",
             "notes": [
-              "Available on global scope from version 43"
+              "From 27to 29, this was only available on service workers."
```
Comment (chrisdavidmills) [inline api/Cache.json]: Typo - add space.

File: api/Cache.json [hunk 3]
```diff
@@ -69,21 +69,39 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Cache/add",
           "support": {
+            "webview_android": {
+              "version_added": "44",
+              "notes": [
+                "Requires HTTPS from version 46"
```
Comment (chrisdavidmills) [inline api/Cache.json]: Period at end of sentence, for consistency with other notes?

File: api/Cache.json [hunk 4]
```diff
@@ -69,21 +69,39 @@
...
             "chrome": {
               "version_added": "44",
               "notes": [
                 "Requires HTTPS from version 46"
               ]
             },
+            "chrome_android": {
+              "version_added": "44",
+              "notes": [
+                "Requires HTTPS from version 46"
```
Comment (chrisdavidmills) [inline api/Cache.json]: Again, add period? Same for other HTTPS notes too - I'll not comment them all.

File: api/Cache.json (hunk 1 above)
Comment (jpmedley) [reply api/Cache.json]: I thought it was required. Sorry.

**Comment by jpmedley** [comment]
@chrisdavidmills PTAL

**Pull request merged**